src/components/FullHeader/FullHeader.js

```diff
@@ -20,10 +20,10 @@ const getSelectedKeys = pathname => {
 	switch (pathname) {
 		case '/marketplace':
 			return '4';
-		case '/clusters':
-			return '2';
```

Comment:
How about we name these keys as `apps`, `clusters`. This would solve the active item issue as well.  Wdyt ?

Reply:
Yes, I agree let me change